packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -530,3 +544,39 @@ class _TooltipOverlay extends StatelessWidget {
     );
   }
 }
+
+class _TooltipSemanticsWidget extends SingleChildRenderObjectWidget {
+  const _TooltipSemanticsWidget({
+    Key key,
+    Widget child
+  }) : super(key: key, child: child);
+
+  @override
+  RenderObject createRenderObject(BuildContext context) {
+    return _RenderTooltipSemanticsWidget();
+  }
+}
+
+class _RenderTooltipSemanticsWidget extends RenderProxyBox {
```

Comment:
Maybe add a quick comment here explaining why we have this widget and what it does to the future reader.